examples/experimental/dagster-airlift/dagster_airlift/core/top_level_dag_def_api.py

```diff
@@ -56,6 +62,47 @@ def assets_def_with_af_metadata(
     )
 
 
+def assets_with_task_mappings(
+    dag_id: str, task_mappings: Mapping[str, Iterable[Union[AssetsDefinition, AssetSpec]]]
+) -> Sequence[Union[AssetsDefinition, AssetSpec]]:
+    """Modify assets to be associated with a particular task in Airlift tooling. Used
+    in concert with `proxying_to_dagster` in dagster_airlift.in_airflow.
```

Comment:
Fine with keeping this language for now, but you really don't need to do that right. You only need to do that at the migrating step.

Reply:
great callout